test/aircraft/FlightManagementSystem/RouteModel.spec.js

```diff
@@ -592,6 +600,86 @@ ava('.hasWaypointName() returns true when the specified waypoint is in the route
     t.true(result);
 });
 
+ava('.isRunwayModelValidForSid() returns false if passed runway is not an instance of RunwayModel', (t) => {
+    const model = new RouteModel(complexRouteStringMock);
+    const airportModel = createAirportModelFixture();
+    const runwayModel = airportModel.getRunway('00');
+    const expectedResult = false;
+    const result = model.isRunwayModelValidForSid(runwayModel);
+
+    t.true(result === expectedResult);
```

Comment:
I am a fan of verbosity, but this is actually redundant and a little more confusing.  You're using `t.true()` with strict equals to verify a value is `false`.  

This matcher could be written:
```js
t.false(result);
// or
t.false(model.isRunwayModelValidForSid(runwayModel));
```

I'd be ok with either and feel free to apply the chosen one to all the other instances like this within this file

Reply:
yeah i guess that is a bit silly. what can i say... i'm a sucker for patterns

fixed!

Reply:
also fixed for instances of 
```
const expectedResult = true;

t.true(result === expectedResult);
```
in favor of
```
const result = kljdhflgkjdhlfg

t.true(result);
```